rails/rails_releases/rails_benchmarks/runner

```diff
@@ -28,4 +32,4 @@ bundle install -j4
 
 echo "Running benchmarks with Rails $RAILS_VERSION"
 cd /ruby-bench-suite/rails/benchmarks
-ruby driver.rb -r 2
+ruby driver.rb -r 2 $PATTERNS
```

Comment:
Lets update the README to reflect this change :smile_cat: https://github.com/ruby-bench/ruby-bench-docker/blob/master/README.md#run-ruby-benchmarks-1